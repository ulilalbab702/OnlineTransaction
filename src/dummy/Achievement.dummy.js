export const Achievement = [
    {
        "Code": "FM-JBYPM",
        "Name": "FMC Jembayan PAMA",
        "Percent": 50,
        "BackLogOpen": 1,
        "BackLogClose": 1,
        "Priority1": 1,
        "Priority2": 0,
        "Priority3": 1,
        "Priority4": 0
    }
]