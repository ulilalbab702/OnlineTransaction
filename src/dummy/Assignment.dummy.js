export const listOfMechanicDummy = [
    {
      "Username": "string",
      "FirstName": "string",
      "LastName": "string",
      "Email": "string",
      "MechanicUT": true,
      "Jobs": [
        {
          "JobType": "string",
          "Total": 0
        }
      ]
    }
  ]
export const jobSummaryDummy = {
  "Date": "2020-06-04T16:18:05.5472055",
  "CreatedBy": "AGUS LISANTO",
  "AssignedBy": "AGUS LISANTO",
  "MechanicLeader": {
      "UserName": "82107038@unitedtractors.com",
      "Name": "SUPRIYADI SUPRIYADI"
  },
  "Mechanics": [],
  "HandoverDate": null,
  "HandoverCreatedBy": null,
  "HandoverAssignedBy": null,
  "HandoverMechanicLeader": null,
  "HandoverMechanics": null,
  "Job": {
      "WoId": "8ff577d1-2911-4bbd-9d55-ffad4fe76fc5",
      "WoNumber": "71018650",
      "JobType": "INS",
      "UnitModel": "HD465-7",
      "SerialNumber": "15163",
      "UnitCode": "HT3022",
      "EquipmentNumber": "71506",
      "IsHandover": false,
      "Id": "a162bd06-6aca-499a-9f97-a5839b8a0612",
      "CreatedBy": "80107002@unitedtractors.com",
      "CreatedDate": "2020-06-04T16:18:05.5472055",
      "UpdatedBy": "80107002@unitedtractors.com",
      "UpdatedDate": "2020-06-04T16:18:05.5472055",
      "IsActive": true
  }
}