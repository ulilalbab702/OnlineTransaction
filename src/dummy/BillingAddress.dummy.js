export const dataBillingAddressDummy = [
    {
        "fullName": "Test Integrasi",
        "contactNumber": "088122345",
        "email": "testintegrasi@unitedtractors.com",
        "position": "string",
        "addressLabel": "Kantor PT Global Service Indonesia",
        "city": "Jakarta",
        "address": "Jl Raya kaliabang Tengah, Ruko San Jaya, Blok A5 No 9,\r\nSebalah Indomaret",
        "postalCode": "13920",
        "locationLatitude": "3.669276961596295",
        "locationLongitude": "102.4003031491451",
        "isPrimary": true
    }]