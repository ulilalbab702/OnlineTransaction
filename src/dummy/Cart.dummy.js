export const cartDummyList = {
    "cartProductResponse": {
        "data": [
            {
                "cartId": "d7e23521-0d23-4f37-aceb-77f8f75b3d16",
                "productId": "32bdd633-932f-465a-b93c-219a70971008",
                "name": "600-311-8291",
                "imageThumbnailUrl": "https://untr-my.sharepoint.com/:i:/g/personal/cst_dev04_unitedtractors_com/EW__sgepd2xJs1m7uxNquHQBbRKmqLZ3DkmR7N1Mb7JWww?e=1f9bRa",
                "qty": 3,
                "priceAfter": 300000,
                "priceBefore": 500000,
                "currency": "IDR",
                "cookiesData": ""
            },
            {
                "cartId": "10d112f9-0a0b-4c26-87b6-87ffb0374a77",
                "productId": "4211034d-8571-4ed6-84b2-8b380443bf7b",
                "name": "568002",
                "imageThumbnailUrl": "https://untr-my.sharepoint.com/:i:/g/personal/cst_dev04_unitedtractors_com/EW__sgepd2xJs1m7uxNquHQBbRKmqLZ3DkmR7N1Mb7JWww?e=1f9bRa",
                "qty": 2,
                "priceAfter": 800000,
                "priceBefore": 1000000,
                "currency": "IDR",
                "cookiesData": ""
            }
        ],
        "meta": {
            "totalItems": 2,
            "pageNumber": 1,
            "pageSize": 5,
            "totalPages": 1,
            "hasPreviousPage": false,
            "hasNextPage": false,
            "nextPageNumber": 1,
            "previousPageNumber": 1
        }
    },
    "subTotalPrice": 2500000
}
