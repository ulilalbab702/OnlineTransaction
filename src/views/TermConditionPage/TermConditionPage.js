import React, { useEffect, useState } from "react";
import {
    Row, CardBody
} from 'reactstrap';
import './style/index.scss';
import Skeleton from 'react-loading-skeleton';
import Breadcrumb from "components/Breadcrumb/Breadcrumb";
import firebase from "../../firebase/firebase";
import { MENU } from "constants/menu";


const TermConditionPage = (props) => {
    const [isLoaded, setIsLoaded] = useState(false)

    const setAnalyticClevertap = (action, event, screen, product) => {
        if (props.user) {
            const { userName } = props.user;
            if (product !== null) {
                window.clevertapEventProduct(`${product[0] + "_LogedIn"}`, product[1]);
            } else {
                window.clevertapEvent(`${event + "_LogedIn"}`, props.user);
            }
            firebase.analytics().setUserId(userName);
            firebase.analytics().setUserProperties(userName, "username:" + userName + `||View: ${screen}`);
            firebase.analytics().setCurrentScreen(screen, screen);
            firebase.analytics().logEvent(`${event + "_LogedIn"}`);
            firebase.analytics().setUserProperties("username", userName);
            firebase.analytics().setUserProperties(action, `${event + "_LogedIn"}`);
        }
    };

    useEffect(() => {
        setAnalyticClevertap("view", "View_TermCondition_Screen", "View_TermCondition_Screen", null);
        window.scrollTo(0, 0);
    }, []);

    useEffect(() => {
        const { tokenResponse: { accessToken } } = props.user;
        async function fetchData() {
            const response = await props.fetchTermCondition(accessToken);
        }
        fetchData();
        if (props.dataTermCondition != undefined) {
            setIsLoaded(true);
        }
    }, []);

    const _renderTextTerm = (data) => {
        return (
            <td style={{ paddingLeft: '-1em' }}>{data.map((item) => {
                return (
                    <div style={{ paddingLeft: '-1em', paddingTop: '0.5em', whiteSpace: 'pre-wrap' }} className="contentParagraph">{item.content}</div>
                )
            })}
            </td>

        )
    }

    const _renderBreadCrumps = () => {
        const breadcrumps = [
            {
                'url': MENU.HOME,
                'name': 'Part Online Transaction'
            }
        ];
        return (
            <Breadcrumb
                linkBreadcrumb={breadcrumps}
                typography={"Syarat & Ketentuan"}
            />
        )
    };

    return (
        <Row className="bgTC">
            <CardBody className="bgContainerTC">
                <div style={{ marginBottom: '1rem' }}>
                    {_renderBreadCrumps()}
                </div>
                <div style={{ textAlign: 'left' }}>
                    <h2 style={{ color: '#D2B520' }} className="tncHeader1"><strong>Syarat dan Ketentuan</strong></h2>
                </div>
                {props.dataTermCondition != undefined ?
                    props.dataTermCondition.termAndConditionHeader != null ?
                        props.dataTermCondition.termAndConditionHeader.map((item) => {
                            return (
                                <>
                                    {item.title != "PEMBUKAAN" ?
                                        <table style={{ marginTop: '2em' }}>
                                            <tr>
                                                <td className="tncHeader2">
                                                    {
                                                        isLoaded ?
                                                            <strong>{item.title}</strong> :
                                                            <Skeleton count={20} />
                                                    }
                                                </td>
                                            </tr>
                                            <tr>
                                                <td className="paddingContent">
                                                    {_renderTextTerm(item.termAndConditionDetail)}
                                                </td>
                                            </tr>

                                        </table>
                                        :
                                        null}
                                </>
                            )
                        })
                        :
                        <div style={{ marginTop: '2em' }}>
                            <Skeleton count={1} />
                            <Skeleton count={30} />
                        </div>
                    :
                    null
                }
            </CardBody>
        </Row>
    )
};
export default TermConditionPage;