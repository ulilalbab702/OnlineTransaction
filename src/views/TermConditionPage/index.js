import TermConditionPage from './TermConditionPage';
import { connect } from 'react-redux';
import { TERM_CONDITION_REQ } from 'actions/actionTypes';
import { login } from 'actions/user';
import { push } from 'connected-react-router';
import {
    createLoadingSelector,
} from 'utils/selector.helper'
import { termConditionGet } from 'actions/termCondition';

//Set selector loading 
const loadingSelector = createLoadingSelector([TERM_CONDITION_REQ]);
const mapStateToProps = state => ({
    ...state.user,
    isLoading: loadingSelector(state),
    dataTermCondition: state.termCondition.termCondition.data,
    loading: state.termCondition.termCondition.loading,
});

const mapDispatchToProps = dispatch => ({
    login: (data) => dispatch(login(data)),
    push: (url) => dispatch(push(url)),
    fetchTermCondition: (token) => dispatch(termConditionGet(token))
});

export default connect(mapStateToProps, mapDispatchToProps)(TermConditionPage);