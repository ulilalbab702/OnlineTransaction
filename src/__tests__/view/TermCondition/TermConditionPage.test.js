import React from "react";
import toJson from "enzyme-to-json";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import TermConditionPage from "views/TermConditionPage/TermConditionPage";
import { dataTermConditionDummy } from "../../../dummy/TermCondition.dummy";

Enzyme.configure({ adapter: new Adapter() });
describe("Term Condition Page Component", () => {
    const wrapper = shallow(
        <TermConditionPage
            dataTermCondition={dataTermConditionDummy}
        />
    );
    describe("Term Condition Page Component", () => {
        it("Should match to snapshot", () => {
            expect(toJson(wrapper)).toMatchSnapshot();
        });
    });
});