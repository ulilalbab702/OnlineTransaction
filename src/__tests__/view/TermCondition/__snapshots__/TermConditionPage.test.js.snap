// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Term Condition Page Component Term Condition Page Component Should match to snapshot 1`] = `
<Row
  className="bgTC"
  tag="div"
  widths={
    Array [
      "xs",
      "sm",
      "md",
      "lg",
      "xl",
    ]
  }
>
  <CardBody
    className="bgContainerTC"
    tag="div"
  >
    <div
      style={
        Object {
          "textAlign": "left",
        }
      }
    >
      <h2
        className="tncHeader1"
        style={
          Object {
            "color": "#D2B520",
          }
        }
      >
        <strong>
          Terms and conditions
        </strong>
      </h2>
    </div>
    <table
      style={
        Object {
          "marginTop": "2em",
        }
      }
    >
      <tr>
        <td
          className="tncHeader2"
        >
          <Skeleton
            circle={false}
            count={20}
            delay={0}
            duration={1.2}
            height={null}
            width={null}
            wrapper={null}
          />
        </td>
      </tr>
      <tr>
        <td
          className="paddingContent"
        >
          <td
            style={
              Object {
                "paddingLeft": "-1em",
              }
            }
          >
            <div
              className="contentParagraph"
              style={
                Object {
                  "paddingLeft": "-1em",
                  "paddingTop": "0.5em",
                  "whiteSpace": "pre-wrap",
                }
              }
            >
              1. PT United Tractors Tbk atau “UT” adalah suatu perseroan terbatas yang menjalankan usaha penjualan alat berat, pengadaan suku cadang dan jasa service alat berat.
            </div>
            <div
              className="contentParagraph"
              style={
                Object {
                  "paddingLeft": "-1em",
                  "paddingTop": "0.5em",
                  "whiteSpace": "pre-wrap",
                }
              }
            >
              2. UT Connect adalah aplikasi dari UT yang dikembangkan untuk memberikan layanan yang terbaik bagi Pengguna. 
            </div>
            <div
              className="contentParagraph"
              style={
                Object {
                  "paddingLeft": "-1em",
                  "paddingTop": "0.5em",
                  "whiteSpace": "pre-wrap",
                }
              }
            >
              3. Syarat dan Ketentuan adalah perjanjian antara Pengguna dan UT yang berisikan seperangkat peraturan yang mengatur hak, kewajiban, tanggung jawab pengguna dan UT, serta tata cara penggunaan sistem layanan UT Connect. 
            </div>
            <div
              className="contentParagraph"
              style={
                Object {
                  "paddingLeft": "-1em",
                  "paddingTop": "0.5em",
                  "whiteSpace": "pre-wrap",
                }
              }
            >
              4. Pengguna adalah pihak yang menggunakan layanan UT Connect, termasuk namun tidak terbatas pada customer UT, calon customer UT ataupun pihak lain yang telah terdaftar di Database Customer UT dan menerima persetujuan dari UT untuk menggunakan dan/atau mengakses UT Connect. 
            </div>
          </td>
        </td>
      </tr>
    </table>
    <table
      style={
        Object {
          "marginTop": "2em",
        }
      }
    >
      <tr>
        <td
          className="tncHeader2"
        >
          <Skeleton
            circle={false}
            count={20}
            delay={0}
            duration={1.2}
            height={null}
            width={null}
            wrapper={null}
          />
        </td>
      </tr>
      <tr>
        <td
          className="paddingContent"
        >
          <td
            style={
              Object {
                "paddingLeft": "-1em",
              }
            }
          >
            <div
              className="contentParagraph"
              style={
                Object {
                  "paddingLeft": "-1em",
                  "paddingTop": "0.5em",
                  "whiteSpace": "pre-wrap",
                }
              }
            >
              Setiap orang yang menggunakan fasilitas yang disediakan dalam UT Connect, maka mereka melakukannya sepenuhnya dengan risiko dan tanggung jawab sendiri dan merupakan pihak yang berwenang, baik secara perorangan maupun badan hukum sesuai dengan Anggaran Dasar Pengguna serta menggunakan UT Connect untuk tujuan yang tidak bertentangan dengan ketentuan-ketentuan hukum yang berlaku. UT tidak bertanggung jawab atas kehilangan atau kerusakan apa pun yang disebabkan oleh apa pun, entah disebabkan oleh, atau terkait dengan mengakses UT Connect ini atau penggunaannya atau upaya pemanfaatan fasilitas yang ditawarkan di sini atau dari tindakan, masalah, atau hal lain apa pun yang terkait dengan UT Connect ini. 
            </div>
            <div
              className="contentParagraph"
              style={
                Object {
                  "paddingLeft": "-1em",
                  "paddingTop": "0.5em",
                  "whiteSpace": "pre-wrap",
                }
              }
            >
              UT dengan upaya terbaiknya akan selalu berupaya untuk menjaga UT Connect dapat berjalan secara aman, nyaman, dan berfungsi dengan baik namun UT tidak menjamin kemungkinan adanya berbagai gangguan terhadap UT Connect sehingga menyebabkan layanan terganggu dan tidak dapat beroperasi terus-menerus. Seluruh informasi dan data yang ditampilkan dalam UT Connect memiliki kemungkinan tidak ditampilkan secara real time. UT dengan tegas menyatakan tidak bertanggung jawab atas kerusakan apapun, termasuk tidak terbatas pada kerugian langsung, tidak langsung, atau ganti rugi atas kehilangan penggunaan, keuntungan, data atau wujud lainnya, kerusakan nama baik atau reputasi, atau biaya pengadaan barang dan jasa, yang timbul dari atau terkait dengan penggunaan, ketidakmampuan untuk menggunakan, kinerja,  atau kegagalan dari UT Connect ini, diantaranya yang diakibatkan secara langsung atau tidak langsung dari:
            </div>
            <div
              className="contentParagraph"
              style={
                Object {
                  "paddingLeft": "-1em",
                  "paddingTop": "0.5em",
                  "whiteSpace": "pre-wrap",
                }
              }
            >
              1. Kesalahanan penggunaan atau ketidakmampuan Pengguna dalam menggunakan layanan UT Connect.
            </div>
            <div
              className="contentParagraph"
              style={
                Object {
                  "paddingLeft": "-1em",
                  "paddingTop": "0.5em",
                  "whiteSpace": "pre-wrap",
                }
              }
            >
              2. Penyalahgunaan layanan UT Connect oleh Pengguna maupun pihak lainnya, baik penggunaan dari perangkat dan/atau akun Penggunanya. 
            </div>
            <div
              className="contentParagraph"
              style={
                Object {
                  "paddingLeft": "-1em",
                  "paddingTop": "0.5em",
                  "whiteSpace": "pre-wrap",
                }
              }
            >
              3. Keterlambatan atau gangguan dalam layanan UT Connect, baik karena adanya gangguan dari luar terhadap sistem maupun karena pemeliharaan, perbaikan (maintenance) atas UT Connect.
            </div>
            <div
              className="contentParagraph"
              style={
                Object {
                  "paddingLeft": "-1em",
                  "paddingTop": "0.5em",
                  "whiteSpace": "pre-wrap",
                }
              }
            >
              4. Kelalaian dan kerugian yang ditimbulkan oleh masing-masing Pengguna berkaitan dengan penggunaan UT Connect miliknya.
            </div>
            <div
              className="contentParagraph"
              style={
                Object {
                  "paddingLeft": "-1em",
                  "paddingTop": "0.5em",
                  "whiteSpace": "pre-wrap",
                }
              }
            >
              5. Pelanggaran Hak atas Kekayaan Intelektual UT yang melekat pada UT Connect.
            </div>
            <div
              className="contentParagraph"
              style={
                Object {
                  "paddingLeft": "-1em",
                  "paddingTop": "0.5em",
                  "whiteSpace": "pre-wrap",
                }
              }
            >
              6. Virus atau perangkat lunak berbahaya lainnya yang diakibatkan oleh penggunaan layanan UT Connect.
            </div>
            <div
              className="contentParagraph"
              style={
                Object {
                  "paddingLeft": "-1em",
                  "paddingTop": "0.5em",
                  "whiteSpace": "pre-wrap",
                }
              }
            >
              7. Gangguan, bug, kesalahan atau ketidakakuratan apapun dalam penggunaan layanan UT Connect.
            </div>
            <div
              className="contentParagraph"
              style={
                Object {
                  "paddingLeft": "-1em",
                  "paddingTop": "0.5em",
                  "whiteSpace": "pre-wrap",
                }
              }
            >
              8. Kerusakan pada perangkat keras Pengguna dikarenakan penggunaan setiap layanan UT Connect oleh Pengguna.
            </div>
            <div
              className="contentParagraph"
              style={
                Object {
                  "paddingLeft": "-1em",
                  "paddingTop": "0.5em",
                  "whiteSpace": "pre-wrap",
                }
              }
            >
              9. Tindakan hukum yang diambil sehubungan dengan penyalahgunaan akun Pengguna.
            </div>
            <div
              className="contentParagraph"
              style={
                Object {
                  "paddingLeft": "-1em",
                  "paddingTop": "0.5em",
                  "whiteSpace": "pre-wrap",
                }
              }
            >
              10. Adanya tindakan peretasan yang dilakukan oleh pihak ketiga kepada akun Pengguna.
            </div>
          </td>
        </td>
      </tr>
    </table>
    <table
      style={
        Object {
          "marginTop": "2em",
        }
      }
    >
      <tr>
        <td
          className="tncHeader2"
        >
          <Skeleton
            circle={false}
            count={20}
            delay={0}
            duration={1.2}
            height={null}
            width={null}
            wrapper={null}
          />
        </td>
      </tr>
      <tr>
        <td
          className="paddingContent"
        >
          <td
            style={
              Object {
                "paddingLeft": "-1em",
              }
            }
          >
            <div
              className="contentParagraph"
              style={
                Object {
                  "paddingLeft": "-1em",
                  "paddingTop": "0.5em",
                  "whiteSpace": "pre-wrap",
                }
              }
            >
              1. UT dengan upaya terbaiknya akan bertanggung jawab dalam melindungi data pribadi Pengguna dan tunduk kepada Hukum yang Berlaku. Kami memiliki komitmen untuk mengelola, melindungi dan memproses data pribadi Pengguna sesuai dengan Syarat dan Ketentuan ini yang akan berlaku terhadap seluruh data pribadi yang kami peroleh dari Pengguna.
            </div>
            <div
              className="contentParagraph"
              style={
                Object {
                  "paddingLeft": "-1em",
                  "paddingTop": "0.5em",
                  "whiteSpace": "pre-wrap",
                }
              }
            >
              2. UT dapat menggunakan dan mengungkapkan data pribadi Pengguna untuk tujuan-tujuan yang diperlukan agar kami dapat menyampaikan produk dan jasa kami kepada Pelanggan, termasuk untuk:
            </div>
            <div
              className="contentParagraph"
              style={
                Object {
                  "paddingLeft": "-1em",
                  "paddingTop": "0.5em",
                  "whiteSpace": "pre-wrap",
                }
              }
            >
              		a. mendaftarkan dan mengelola user account atau akun Pengguna;
            </div>
            <div
              className="contentParagraph"
              style={
                Object {
                  "paddingLeft": "-1em",
                  "paddingTop": "0.5em",
                  "whiteSpace": "pre-wrap",
                }
              }
            >
              		b. mengkomunikasikan kepada Pengguna hal-hal yang berkaitan dengan perubahan material pada UT Connect, perubahan Kebijakan UT, perubahan persyaratan dan ketentuan lainnya, maupun perubahan lain berkaitan dengan pengelolaan akun Pengguna;
            </div>
            <div
              className="contentParagraph"
              style={
                Object {
                  "paddingLeft": "-1em",
                  "paddingTop": "0.5em",
                  "whiteSpace": "pre-wrap",
                }
              }
            >
              		c. melakukan personalisasi dan meningkatkan customer experience Pengguna saat Pengguna mengunjungi UT Connect;
            </div>
            <div
              className="contentParagraph"
              style={
                Object {
                  "paddingLeft": "-1em",
                  "paddingTop": "0.5em",
                  "whiteSpace": "pre-wrap",
                }
              }
            >
              		d. memonitor dan memastikan keberlakuan Syarat dan Ketentuan Jasa;
            </div>
            <div
              className="contentParagraph"
              style={
                Object {
                  "paddingLeft": "-1em",
                  "paddingTop": "0.5em",
                  "whiteSpace": "pre-wrap",
                }
              }
            >
              		e. memastikan UT Connect berfungsi dengan baik dan untuk meningkatkan kinerja UT dengan melakukan beberapa aktifitas tertentu yang diperlukan, seperti melakukan debugging, analisa statistik untuk mengoptimalkan UT Connect; 
            </div>
            <div
              className="contentParagraph"
              style={
                Object {
                  "paddingLeft": "-1em",
                  "paddingTop": "0.5em",
                  "whiteSpace": "pre-wrap",
                }
              }
            >
              		f. melakukan pembaharuan UT Connect, termasuk dalam rangka meningkatkan keamanan akun pengguna.
            </div>
            <div
              className="contentParagraph"
              style={
                Object {
                  "paddingLeft": "-1em",
                  "paddingTop": "0.5em",
                  "whiteSpace": "pre-wrap",
                }
              }
            />
            <div
              className="contentParagraph"
              style={
                Object {
                  "paddingLeft": "-1em",
                  "paddingTop": "0.5em",
                  "whiteSpace": "pre-wrap",
                }
              }
            >
              3. Pengguna mengakui dan menyetujui bahwa UT berhak untuk mengungkapkan data pribadi Pengguna kepada institusi pemerintah, pengawas pemerintah, pajak, penegak hukum atau pihak berwenang lain, jika UT memiliki dasar yang wajar untuk percaya bahwa pengungkapan data pribadi Pengguna diperlukan untuk tujuan memenuhi kewajiban, persyaratan, atau ketentuan apa pun, baik secara sukarela atau wajib, sebagai akibat menaati perintah, bekerja sama dalam penyelidikan, dan/atau permintaan yang bersifat apa pun juga dengan pihak-pihak tersebut. Sejauh diizinkan oleh undang-undang yang berlaku, Pengguna setuju untuk tidak mengambil tindakan apapun dan/atau guna melepaskan hak-hak Pengguna untuk mengambil tindakan apapun terhadap UT terkait dengan pengungkapan data pribadi Pengguna.
            </div>
            <div
              className="contentParagraph"
              style={
                Object {
                  "paddingLeft": "-1em",
                  "paddingTop": "0.5em",
                  "whiteSpace": "pre-wrap",
                }
              }
            />
            <div
              className="contentParagraph"
              style={
                Object {
                  "paddingLeft": "-1em",
                  "paddingTop": "0.5em",
                  "whiteSpace": "pre-wrap",
                }
              }
            >
              4. Pengguna diwajibkan melindungi akun UT Connect miliknya, termasuk tetapi tidak terbatas pada menjaga kerahasiaan sandi Pengguna. Pengguna bertanggung jawab atas aktivitas yang terjadi pada atau melalui akun UT Connect miliknya. Apabila Pengguna mendapati adanya penggunaan akun maupun sandi UT Connect yang anda miliki yang patut dicurigai, mohon untuk segera melaporkan kepada UT.
            </div>
          </td>
        </td>
      </tr>
    </table>
  </CardBody>
</Row>
`;
